designer/server/src/test/scala/pl/touk/nussknacker/test/mock/MockDeploymentManager.scala

```diff
@@ -75,16 +78,33 @@ class MockDeploymentManager private (
       case None          => super.processStateDefinitionManager
     }
 
-  override def getProcessStates(
-      name: ProcessName
-  )(implicit freshnessPolicy: DataFreshnessPolicy): Future[WithDataFreshnessStatus[List[StatusDetails]]] = {
+  override protected def getScenarioDeploymentsStatusesWithJobOverview(
+      scenarioName: ProcessName
+  )(
+      implicit freshnessPolicy: DataFreshnessPolicy
+  ): Future[WithDataFreshnessStatus[List[(DeploymentStatusDetails, JobOverview)]]] = {
```

Comment:
This method now takes `JobOverview`. It was needed to remove some flink-local, not used anywhere else fields from `DeploymentStatusDetails` (old `StatusDetails`). Finally, we should resolve TODO on top of this class, and use `MockableDeploymentManager` instead of this class